
# message to set pid constants

uint8 dof

float32 kp
float32 ki
float32 kd

float32 acceptable_error

float32 ko
